hivemind/client/averaging/matchmaking.py

```diff
@@ -462,5 +462,13 @@ async def _declare_averager_periodically(self, key_manager: GroupKeyManager):
                                                        looking_for_group=False)
 
 
+def compute_schema_hash(tensors: Sequence[torch.Tensor]) -> bytes:
```

Comment:
In the master branch, `compute_schema_hash` is already defined in `hivemind/client/averaging/__init__.py`. Please either remove this change or remove the previous definition from `__init__.py`.